# Descriptions are capitalized
#### `graphql-schema/descriptions-are-capitalized`

## Rule details

Validates that all field arguments have a description.

## Default configuration

```json
{
    "overrides":[
        {
            "rules": {
               "graphql-schema/descriptions-are-capitalized": ["warn"]
            }
        }
    ]
}
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
