# Enum values are all capitalized
#### `graphql-schema/enum-values-all-caps`

## Rule details

Validates that all enum values are capitalized.

## Default configuration

```json
{
    "overrides":[
        {
            "rules": {
               "graphql-schema/enum-values-all-caps": ["error"]
            }
        }
    ]
}
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
