# Enum values sorted alphabetically
#### `graphql-schema/enum-values-sorted-alphabetically`

## Rule details

Validates that all enum values sorted alphabetically.

## Default configuration

```json
{
    "overrides":[
        {
            "rules": {
               "graphql-schema/defined-types-are-used": ["warn"]
            }
        }
    ]
}
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
