# Input object values have descriptions
#### `graphql-schema/input-object-values-have-descriptions`

## Rule details

* Validates that input object values have a description.

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
