# Input object values are camel cased
#### `graphql-schema/input-object-values-are-camel-cased`

## Rule details

* Validates  that input object value names are camel cased.

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
