# Input object fields sorted alphabetically
#### `graphql-schema/input-object-fields-sorted-alphabetically`

## Rule details

* Validates that all input object fields sorted alphabetically. 

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
