# Types have descriptions
#### `graphql-schema/types-have-descriptions`

## Rule details

* Validates that interface types, object types, union types, scalar types, enum types and input types have descriptions.

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
