# Type fields have descriptions
#### `graphql-schema/type-fields-have-descriptions`

## Rule details

* Validates that object type fields and interface type fields have a description.

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
