# Type fields sorted alphabetically
#### `graphql-schema/type-fields-sorted-alphabetically`

## Rule details

* Validates  that all type object fields sorted alphabetically.

## Default configuration
```json
```

## Examples

### valid
```graphql
```

### invalid
```graphql
```
